index.bs

```diff
@@ -193,6 +193,75 @@ The term <dfn id="UnixEpoch">Unix Epoch</dfn> is used in this specification to r
 (or 1970-01-01T00:00:00Z ISO 8601);
 this is the same time that is conceptually "<code>0</code>" in ECMA-262 [[ECMA-262]].
 
+<div algorithm>
+The <dfn id="slice-blob">slice blob</dfn> algorithm given a {{Blob}} <var>blob</var>,
```

Comment:
You probably want this to be exported so you can reference it from fetch? And this should probably be an abstract-op, and I don't think it needs an explicit ID.

```suggestion
The <dfn abstract-op export>slice blob</dfn> algorithm given a {{Blob}} <var>blob</var>,
```

Reply:
It should not be an abstract-op (that's mainly JS stuff, all our algorithms are the default dfn most of the time), but you're right that relying on the generated ID is fine.